test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -449,3 +449,32 @@
       initNodes: 5000
       initPods: 2000
       measurePods: 5000
+
+- name: PendingPodsWithMixedChurn
+  workloadTemplate:
+  - opcode: createNodes
+    countParam: $initNodes
+  - opcode: createPods
+    countParam: $initPods
+    podTemplatePath: config/pod-high-priority-large-cpu.yaml
+    skipWaitToCompletion: true
+  - opcode: churn
```

Comment:
should the churn pods be in the same namespace as the createPods op? I can't remember if that makes a difference in the scheduler

Reply:
I assume the users can provide namespace name in the configuration if they think namespace is correlated.